index.html

```diff
@@ -0,0 +1,122 @@
+            </ul>
+        </div>
+    </nav>
+    <nav class="second-nav fixed-top-nav" id="first-header">
+        <div class="container">
+            <i class="fa-solid fa-bars" id="menu-image"></i>
+            <div class="navigation">
+                <img class="navigation-logo" src="assets/musicFestivalLogo.png" width="65px" alt="Festival">
+                <ul class="second-nav-ul float-right">
+                    <li><a class="second-nav-i" href="about.html">About</a></li><br>
+                    <li><a class="second-nav-i" href="#program">Program</a></li><br>
+                    <li><a class="second-nav-i" href="#join">Join</a></li><br>
+                    <li><a class="second-nav-i" href="#sponsor">Sponsor</a></li><br>
+                    <li><a class="second-nav-i" href="#updates">News</a></li><br>
+                    <li><a href="#" class="mt-3 p-2" id="cc-btn">CC Campaign</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <nav class="mobile-nav">
+        <span class="cross">x</span>
+        <ul class="mobile-nav-ul">
+            <li><a class="mobile-nav-i" href="about.html">About</a></li><br>
+            <li><a class="mobile-nav-i" href="#program">Program</a></li><br>
+            <li><a class="mobile-nav-i" href="#join">Join</a></li><br>
+            <li><a class="mobile-nav-i" href="#sponsor">Sponsor</a></li><br>
+            <li><a class="mobile-nav-i" href="#updates">News</a></li><br>
+            <li><a href="#" class="mt-3 p-2" id="cc-btn">CC Campaign</a></li><br>
+        </ul>
+    </nav>
+    <main class="top-section">
+        <section class="top-bg">
+            <section class="hero container"><br><br><br><br>
+                <h2 class="salute">"Hello! Sharing World"</h2>
+                <h1 class="hero-first-title">Kings of Music<br>Festival</h1><br>
+            </section>
+            <section class="text-message">
+                <div class="container">
+                    <p class="message-p p-3 border border-white">Grab your friends, grab your dancing shoes and let's go! Let's make this festival an unforgettable experience!</p>
+                    <h2 class="message-time">📆23 June(Fri) to 25 June(Sun) 2023</h2>
+                    <p>📍108 Holland Drive, Malibu.</p>
+                </div>
+            </section><br><br><br>
+        </section>
+
+        <section class="program" id="program">
+            <br>
+            <h4 class="text-center section-title-white">Main Program</h4>
+            <hr class="line-name">
+            <div class="program-container container">
+
+
+            </div>
+            <br>
+            <div class="container text-center">
+                <a href="#" class="ticket-btn p-3" id="ticket">Get Your Ticket Now</a>
+                <a href="#" class="link-to-program p-3">See The Whole Program</a>
+            </div><br><br>
+        </section>
+
+        <section class="first-speaker">
```

Comment:
Excellent work implementing dynamically your Featured Speakers section 💯 

There are some changes to make here to make this a better match with Behances design

- [ ] I can see that your font-size is too big for this section and maybe that it's using all the space on the cards, maybe you can reduce it a bit to use the space for each element

- [ ] Kindly fix the size of the images to have the same size between them. I've seen that you are giving a size in the properties, but maybe the image is different in your assets.

- [ ] As we can see, here the font-family used is `Lato`, kindly implement it

- [ ] There's a bit more space between the grey line and the subtitle and description, kindly add this missing space


![image](https://github.com/Tarto-4/First-Capstone/assets/93630700/0110e9e6-d656-41cd-a66d-926b833fad54)

![image](https://github.com/Tarto-4/First-Capstone/assets/93630700/af3fad02-4035-4b13-a1d4-9f31f8f6f45d)